client/my-sites/site-settings/form-writing.jsx

```diff
@@ -62,26 +62,32 @@ class SiteSettingsFormWriting extends Component {
 			uniqueEventTracker,
 			fields,
 			handleSelect,
+			handleSubmitForm,
 			handleToggle,
 			handleAutosavingToggle,
 			handleAutosavingRadio,
+			isJetpackSite: _isJetpackSite,
```

Comment:
Nit: prefixing variables with underscores doesn't comply to our [coding standards](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#naming-conventions). How about naming that one `isSiteJetpack` or `isJetpack` or `siteIsJetpack` early in `mapStateToProps`?